functions/interface.php

```diff
@@ -107,7 +106,7 @@ function admin_notices() {
 
 				<?php foreach ( $notices['server_config'] as $notice ) : ?>
 
-					<li><?php echo wp_kses_data( $notice ); ?></li>
+					<li><?php echo print_whitelist_html( $notice ); ?></li>
```

Comment:
and `a`? There are lots of cases where we refer either to BWP docs or PHP docs. However, in some cases we want to allow URL to be localised and in some now. How do we tackle that?